app/src/pages/common/settingsPage/autoAnalysisTab/analysisForm/strategyBlock/strategyBlock.jsx

```diff
@@ -173,6 +228,41 @@ export class StrategyBlock extends Component {
             </span>
           </InputRadio>
         </FormField>
+
+        <FormField
+          fieldWrapperClassName={cx('switcher-wrapper')}
+          label={formatMessage(messages.uniqueErrorAutoAnalysisSwitcherTitle)}
+          customBlock={{
+            node: <p>{Parser(formatMessage(messages.uniqueErrorAutoAnalysisStatusInfo))}</p>,
+          }}
+          withoutProvider
+        >
+          <InputBigSwitcher
+            value={data[UNIQUE_ERROR_ENABLED]}
+            onChange={this.changeUniqueErrorAutoAnalysisEnabled}
+            disabled={uniqueErrorAutoAnalysisDisabled}
+            mobileDisabled
+            title={tooltip}
+          />
+        </FormField>
+
+        <FormField
+          fieldWrapperClassName={cx('unique-errors-remove-numbers')}
+          label={formatMessage(importMessages.uniqueErrAnalyzeModalFieldName)}
+          withoutProvider
+        >
+          <InputDropdown
+            value={data[UNIQUE_ERROR_REMOVE_NUMBERS]}
+            options={options}
+            disabled={
+              uniqueErrorsRemoveNumbersDisabled ||
+              uniqueErrorAutoAnalysisDisabled ||
+              !data[UNIQUE_ERROR_ENABLED]
```

Comment:
Same here. Please discuss it with team lead or BA, should be disabled dropdown in case of service analyze is enabled but Unique error auto-analysis is off

Reply:
I will clarify.